pkg/question/question.go

```diff
@@ -1189,3 +1200,37 @@ func AskTerminationConfirmation(instanceIds []string) string {
 
 	return answer
 }
+
+func AskCapacityType(instanceType string) string {
+	ec2Pricing := ec2pricing.New(session.New())
+	onDemandPrice, err := ec2Pricing.GetOnDemandInstanceTypeCost(instanceType)
+	formattedOnDemandPrice := ""
+	if err == nil {
+		onDemandPrice = math.Round(onDemandPrice*10000) / 10000
+		formattedOnDemandPrice = fmt.Sprintf("($%s/hr)", strconv.FormatFloat(onDemandPrice, 'f', -1, 64))
+	}
+
+	spotPrice, err := ec2Pricing.GetSpotInstanceTypeNDayAvgCost(instanceType, []string{}, 1)
+	formattedSpotPrice := ""
+	if err == nil {
+		spotPrice = math.Round(spotPrice*10000) / 10000
+		formattedSpotPrice = fmt.Sprintf("($%s/hr)", strconv.FormatFloat(spotPrice, 'f', -1, 64))
+	}
+
+	question := fmt.Sprintf("Select capacity type. Spot instances are available at up to a 90%% discount compared to On-Demand instances,\n" +
+		"but they may get interrupted by EC2 with a 2-minute warning")
+
+	defaultInstanceTypeText := DefaultCapacityTypeText.OnDemand
+	optionsText := fmt.Sprintf("1. On-Demand%s\n2. Spot%s\n", formattedOnDemandPrice,
```

Comment:
Let's add a space between the capacity name and the price:

```suggestion
	optionsText := fmt.Sprintf("1. On-Demand %s\n2. Spot %s\n", formattedOnDemandPrice,
```